fud/fud/stages/verilator/stage.py

```diff
@@ -165,10 +162,11 @@ def cleanup(tmpdir: SourceType.Directory):
         # Schedule
         tmpdir = mktmp()
         # if we need to, convert dynamically sourced json to dat
-        if self.data_path is None:
+        if data_path is None:
             check_verilog_for_mem_read(input_data)
         else:
-            json_to_dat(tmpdir, Source(Path(self.data_path), SourceType.Path))
+            json_to_dat(tmpdir, Source.path(data_path))
+        print(input_data)
```

Comment:
Is this intentional?

Reply:
Nope, just some debugging code. Good catch!